corehq/apps/app_manager/models.py

```diff
@@ -4068,7 +4069,10 @@ def key_server_url(self):
 
     @absolute_url_property
     def ota_restore_url(self):
-        return reverse('corehq.apps.ota.views.restore', args=[self.domain])
+        if use_app_aware_sync(self):
+            # Pass app ID in URL to avoid changing mobile code (CommCareRestorer.java) to pass as GET param
+            return reverse('app_aware_restore', args=[self.domain, self._id])
```

Comment:
(I think it needs to be the latter or will break if any mobile UCR configs change)